layouts/partials/home-page/book-tickets.html

```diff
@@ -1,27 +1,61 @@
 <section class="booking-section">
   <div class="container booking-container">
     <div class="booking-blocks">
-      <div class="booking-banner">
-        <div class="booking-banner-text">
-
-          <div>
-            <div class="booking-leader">
-              <img src="/img/home/booking/check.svg" alt="check"/><span>OWASP Leader / Member</span>
-            </div>
-            <div class="booking-students">
-              <img src="/img/home/booking/check.svg" alt="check"/><span>Students & LLHS Members</span>
-            </div>
+     <div class="booking-info">
+      <h3 class="booking-subtitle booking-subtitle-upper">Time is running out</h3>
+      <h2 class="booking-title">Book your ticket</h2>
+      <p class="booking-text">Lorem Ipsum has been the industry's standard dummy text ever 
```

Comment:
this text doesn't look right 

@olegtaranenko is this going to be a live page?

Reply:
Of course not. I will ask in the slack